resources/metadata/influx.yaml

```diff
@@ -0,0 +1,66 @@
+metadata:
+  name: influxWriteData
+  description: Writes metrics to influxdb
+  longDescription: |
+    In this step, the metrics are written to the timeseries database [InfluxDB](https://www.influxdata.com/time-series-platform/influxdb/)
+spec:
+  inputs:
+    secrets:
+      - name: influxTokenId
+        description: Influxdb token for authentication to the InfluxDB. In 1.8 version use 'username:password' instead.
+        type: jenkins
+    params:
+      - name: serverUrl
+        type: string
+        description: Base URL to the InfluxDB server
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+          - GENERAL
+        mandatory: true
+      - name: authToken
+        type: string
+        description: Token to authenticate to the Influxdb
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+        secret: true
+        resourceRef:
+          - name: influxAuthTokenId
+            type: secret
+      - name: bucket
+        type: string
+        description: Name of database (1.8) or bucket (2.0)
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        default: "piper"
+      - name: organization
+        type: string
+        description: Name of influx organization. Only for Influxdb 2.0
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: dataMap
+        type: "map[string]interface{}"
```

Comment:
since it should be able to pass the parameter also via command line I suggest to switch type to `string` and accept a `json` value